src/natcap/invest/ui/delineateit.py

```diff
@@ -21,6 +21,18 @@ def __init__(self):
                 "for each cell."),
             validator=self.validator)
         self.add_input(self.dem_path)
+
+        self.detect_pour_points = inputs.Checkbox(
+            args_key='detect_pour_points',
+            helptext=(
+                'If this box is checked, run pour point detection algorithm. '
+                'Use detected pour points instead of outlet geometries.'),
+            label='Detect pour points'
+        )
+        self.add_input(self.detect_pour_points)
+        self.detect_pour_points.sufficiency_changed.connect(
+            self.on_detect_pour_points_changed)
```

Comment:
Could you be sure to add this new input to the `self.assemble_args` method below?  Without it, the user's selection of whether to compute pour points won't be passed to the model.